setup.py

```diff
@@ -25,7 +26,26 @@
         'sdk.txt',
     ), session='None')
 
-VERSION = environ.get('TRAVIS_TAG')
+# Get path to version file
+version_path = join(
+    dirname(abspath(__file__)),
+    'VERSION'
+)
+
+# Try to write version file from Travis tag
+travis_tag = environ.get('TRAVIS_TAG')
```

Comment:
IMHO - too complex, and not really fair 

There should be no reference to TRAVIS_TAG in package on pypi (connect-sdk-15.0.tar.gz)
Probably, best solution to write down version into setup.py on build on travis
like:
in git - setup.py looks like:
...
setup(
...
version='0.0.0',
...

in 
.travis.yaml 

script:
- sed -i "s/version = '0.0.0'/version = '${TRAVIS_TAG}'/g" setup.py

Then, setup.py will be modified only while travis build and then package will have just right constant in setup.py
Also, you can also modify connect/__init__.py same way (put __version__ = '0.0.0' in git here)

Reply:
as above solution you can check V2 in https://stackoverflow.com/questions/48943503/travis-pypi-package-version

also, probably, it worth to use pbr (V1 by link above)

Reply:
probably, better to use 15.dev0 to match python version conventions

Reply:
Uploaded commit removing TRAVIS_TAG from Python code and using sed. Do we need PBR then? This solves the problem already and does not imply adding an additional dependency.